stores/amazon.py

```diff
@@ -551,6 +551,10 @@ def checkout(self, test):
         self.wait_for_order_completed(test)
 
         log.info("Order Placed.")
-        self.driver.save_screenshot("screenshot.png")
+        self.save_screenshot()
         self.notification_handler.send_notification("Order Placed", True)
         return True
+
+    def save_screenshot(self):
+        self.driver.save_screenshot(f"screenshot-{os.getpid()}-{time.time()}.png")
+
```

Comment:
This will break notifications as we rely on it being called screenshot.png. Maybe some logic in the notification handler, or renaming the file after the notification is sent.